app/src/test/java/org/p2p/wallet/utils/JvmDecimalFormatter.kt

```diff
@@ -0,0 +1,24 @@
+package org.p2p.wallet.utils
+
+import java.math.RoundingMode
+import java.text.DecimalFormat
+import java.text.DecimalFormatSymbols
+import java.util.Locale
+
+private const val DECIMAL_FORMAT = "###,###."
+
+object JvmDecimalFormatter {
```

Comment:
it's used for mock formatting, as original DecimalFormatter contains android variant with libicu